Prometheus.AspNetCore/Prometheus.AspNetCore.csproj

```diff
@@ -37,7 +37,7 @@
   </ItemGroup>
   
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.AspNetCore" Version="2.1.7" />
+    <PackageReference Include="Microsoft.AspNetCore" Version="2.2.0" />
```

Comment:
IEndpointFeature and in general endpoint routing is only available from ASP.NET Core 2.2. As I wasn't sure if you want to keep the ASP.NET version in sync across all projects, I've updated it everywhere

Reply:
The code intentionally targets 2.1 because that is the v2 LTS version, with 2.2 being end of life (ASP.NET Core and .NET Core are versioned together).

If the minimum requirement is 2.2 for this then this feature should target 3.1 instead (with an `#if NETCOREAPP3_1`).

Reply:
Reverted 2.2 version bump in favor of `#if NETCOREAPP3_1`